fiat-api/src/test/groovy/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluatorSpec.groovy

```diff
@@ -251,6 +251,26 @@ class FiatPermissionEvaluatorSpec extends Specification {
     resourceType << ResourceType.values()*.toString()
   }
 
+  def "should support isAdmin check for a user"() {
+    when:
+    1 * fiatService.getUserPermission("testUser") >> {
+      return new UserPermission.View()
+          .setApplications(Collections.emptySet())
+          .setAdmin(true)
+    }
+    then:
+    evaluator.isAdmin(authentication)
```

Comment:
believe you've got your when and then backward ... at least how I would normally write a test like this.

Alternatively:

```
given:
1 * fiatService.getUserPermission("testUser") >> {
      return new UserPermission.View()
          .setApplications(Collections.emptySet())
          .setAdmin(isAdmin)
    }

expect:
evaluator.isAdmin(authentication) == expectedIsAdmin

where:
isAdmin || expectedIsAdmin
false   || false
true    || true
```

Reply:
:+1: